streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -235,25 +235,27 @@ public void shouldReduceWindowed() throws Exception {
                 .thenComparing(KeyValueTimestamp::value);
 
         windowedOutput.sort(comparator);
-        final long firstBatchWindow = firstBatchTimestamp / 500 * 500;
-        final long secondBatchWindow = secondBatchTimestamp / 500 * 500;
+        final long firstBatchWindowStart = firstBatchTimestamp / 500 * 500;
+        final long firstBatchWindowEnd = firstBatchWindowStart + 500;
```

Comment:
Why don't you use `timeDifference` like you did below?

Reply:
These are time windows so it would be `windowSize`, but I didn't write this test, just updated it to fit with the updated deserializer. I can switch it out to `windowSize` if you think it would help with readability, WYDT?

Reply:
It would probably be better practice but it's not necessary to fix. So if you feel like it